src/main/webapp/app/admin/user-management/user-management.component.ts

```diff
@@ -419,6 +423,58 @@ export class UserManagementComponent implements OnInit, OnDestroy {
         });
     }
 
+    /**
+     * Selects/Unselects all (filtered) users.
+     */
+    selectAllUsers() {
```

Comment:
```suggestion
    toggleAllUserSelection() {
```
I'd change the name to something like this to better reflect the actual behavior of the method